source/extensions/filters/network/common/redis/client_impl.cc

```diff
@@ -94,7 +98,17 @@ PoolRequest* ClientImpl::makeRequest(const RespValue& request, PoolCallbacks& ca
 
   const bool empty_buffer = encoder_buffer_.length() == 0;
 
-  pending_requests_.emplace_back(*this, callbacks);
+  Stats::StatName stat_name;
```

Comment:
nit: you are using 'stat_name' here and in a few other places now as the local variable name indicate the StatName form of a command. I'd prefer to call this variable 'command' as you had it before, or command_stat_name if you prefer. This is because you have tons of other StatName member variables so I think it's better to be explicit about what this particular StatName is.

Reply:
Yeah, I was half and half on this. Makes sense to use `command` to distinguish it from the others.